app/controllers/api/v1/settings_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class SettingsController < Api::V1::ApiController
+      skip_before_action :authenticate_user!
+
+      def need_update
+        current_version = params[:device_version].to_f
+        min_version = Setting.find_by(key: 'min_version').value.to_f
```

Comment:
What happens if min_version is `0.0.1`? Does `to_f` work with it?
Maybe you can use Gem::Version.new(version)